pyscriptjs/src/components/pyscript.ts

```diff
@@ -9,6 +9,8 @@ const logger = getLogger('py-script');
 
 export function make_PyScript(runtime: Runtime) {
     class PyScript extends HTMLElement {
+        srcCode: string
```

Comment:
This could become

```suggestion
        // Save innerHTML information in srcCode so we can access it
        // once connectedCallback cleans innerHTML
        srcCode: string = this.innerHTML
```

So we don't need to do the assignment on line 24, I've added the comment there because it's not clean why we are doing this 😄

Reply:
Yeah, so, there is a subtle difference.
If you notice the first commit of this PR, I do it inside the `constructor`.
But that doesn't work and is kinda wrong. Because when you set `innerHTML` later aka after declaration of the tag.
The assignment to `srcCode` has already happened i.e. `''`

While we want it to get assigned on the evaluation stage and not the declaration stage.
Thus, the assignment is inside `connectedCallback`

This is the reason why the test related to `dynamic tag` started failing.
I will add comments though.